data/constants.js

```diff
@@ -0,0 +1,16 @@
+const PORT = process.env.PORT || 3000;
+
+const SERVER_URL = `http://localhost:${PORT}`;
+
+const COURSES = [
```

Comment:
I mean move only const COURSES to the data folder. PORT and SERVER URL u can save in constants file in utils.

So, courses it's your data for response of your API.
PORT and SERVER_URL are configuration/constant variables.

Reply:
Right, that makes more sense. Thank you.